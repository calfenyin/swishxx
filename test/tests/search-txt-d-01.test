search | | -d | year | 0
