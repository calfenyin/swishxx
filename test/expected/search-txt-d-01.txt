2 502 ./Alice's_Adventures_in_Wonderland.txt 147773 Alice's_Adventures_in_Wonderland.txt
18 528 ./Christmas_Carol,_A.txt 162261 Christmas_Carol,_A.txt
3 404 ./Time_Machine,_The.txt 182203 Time_Machine,_The.txt

